core/include/initcall.h

```diff
@@ -101,6 +101,31 @@ struct initcall {
 
 #define boot_final(fn)			__define_initcall(final, 1, fn)
 
+/*
+ * These nex_* init-calls are provided for drivers and services that
+ * resides in the nexus in case of virtualization.  The init-calls are
```

Comment:
s/resides/reside/ : already addressed (and commit acked) in https://github.com/OP-TEE/optee_os/pull/6686

Reply:
I'll rebase on master to get rid of the "core: add nex_*init-calls" commit.